scripts/test_example_projects_ci.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+SCRIPT_LOCATION=$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")
+
+source ${SCRIPT_LOCATION}/helper_functions.sh
+
+set -e
+
+if [ ! -e 'gradle.properties' ]; then
+    echo "Script was not executed in root directory"
+    exit 1
+fi
+
+
+VERSION="1.0-t"
+
+updateVersionForAllSubprojects $VERSION
+
+./gradlew clean publishToMavenLocal
```

Comment:
technically this file does not test the example projects. So a more appropriate name might be `local_release_with_version`

You could add a bash comment within the file stating that this is for testing example projects